lib/elixir/lib/code/formatter.ex

```diff
@@ -79,7 +80,25 @@ defmodule Code.Formatter do
     # Testing
     all: :*,
     assert: 1,
-    assert: 2
+    assert: 2,
+    assert_in_delta: 3,
+    assert_in_delta: 4,
+    assert_raise: 2,
+    assert_raise: 3,
+    assert_receive: 1,
+    assert_receive: 2,
+    assert_receive: 3,
+    assert_received: 1,
+    assert_received: 2,
+    refute: 1,
+    refute: 2,
+    refute_in_delta: 3,
+    refute_in_delta: 4,
+    refute_receive: 1,
+    refute_receive: 2,
+    refute_receive: 3,
+    refute_received: 1,
+    refute_received: 2
```

Comment:
Oh boy I never pushed these D: I have them still on my local checkout of the formatter 😄